api/util/metrics.js

```diff
@@ -25,33 +25,38 @@ class MetricsHandler {
 	  labelNames: ['type'],
 	});
 
-	  totalMessagesSent = new client.Counter({
-	    name: 'total_messages_sent',
-	    help: 'Total number of messages sent'
-	  });
-
-	  currentConnectionsOpen = new client.Gauge({
-	    name: 'current_connections_open',
-	    help: 'Total number of connections open',
-	    labelNames: ['id']
-	  });
-
-	  totalChatMessagesPerChatRoom = new client.Counter({
-	    name: 'total_chat_messages_per_chatroom',
-	    help: 'Total number of messages sent per chatroom',
-	    labelNames: ['id']
-	  });
-
-	  constructor() {
-	  register.setDefaultLabels({
-	    app: 'sce-core',
-	  });
-	  client.collectDefaultMetrics({ register });
-
-	  Object.keys(this).forEach(metric => {
-	    register.registerMetric(this[metric]);
-	  });
-	  }
+  totalMessagesSent = new client.Counter({
+    name: 'total_messages_sent',
+    help: 'Total number of messages sent'
+  });
+
+  currentConnectionsOpen = new client.Gauge({
+    name: 'current_connections_open',
+    help: 'Total number of connections open',
+    labelNames: ['id']
+  });
+
+  totalChatMessagesPerChatRoom = new client.Counter({
+    name: 'total_chat_messages_per_chatroom',
+    help: 'Total number of messages sent per chatroom',
+    labelNames: ['id']
+  });
+
+  currentSizeOfPrintingFolder = new client.Gauge({
+    name: 'current_size_of_printing_folder',
+    help: 'Current size of printing folder'
+  });
```

Comment:
add the unit of measure to this, like bytes if its bytes. does fs return IEC bytes? if so, add the exact unit used in the help message, helps a lot
```suggestion
  currentSizeOfPrintingFolderBytess = new client.Gauge({
    name: 'current_size_of_printing_folder_bytes',
    help: 'Current size of printing folder'
  });0
```